Lib/test/test_urllib.py

```diff
@@ -1503,6 +1509,30 @@ def test_url2pathname(self):
         self.assertEqual(fn('//localhost/foo/bar'), f'{sep}foo{sep}bar')
         self.assertEqual(fn('///foo/bar'), f'{sep}foo{sep}bar')
         self.assertEqual(fn('////foo/bar'), f'{sep}{sep}foo{sep}bar')
+        self.assertEqual(fn('data:blah'), 'data:blah')
```

Comment:
Just a suggestion that you can use either here or in a follow-up PR, but you could write the tests as follows:

```py
for path, expect in [
	('file:', ''),
	...
]:
	with self.subTest(path=path):
		self.assertEqual(fn(path, require_scheme=True), expect)
```

Could save you a bit of horizontal width but it's up to you (you can also use `check = self.assertEqual` and use `check(fn(...), ...)`; that's what we do in uuid.py because we're always checking many times that something is equal to something else).